packages/EtendoHookBinder/src/hooks/useToolbar.ts

```diff
@@ -0,0 +1,51 @@
+import { useCallback, useEffect, useState } from 'react';
+import { Metadata } from '../api/metadata';
+
+interface ToolbarResponse {
+  response: {
+    buttons: Array<{
```

Comment:
This type looks quite similar to ToolbarButton, maybe you could extend that interface?
Like:

```ts
interface ToolbarButtonResponse extends ToolbarButton {
  icon: string;
}

interface ToolbarResponse {
  response: {
    buttons: ToolbarButtonResponse[];

```